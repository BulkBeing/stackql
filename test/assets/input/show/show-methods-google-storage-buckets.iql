SHOW METHODS 
from google.storage.buckets
;