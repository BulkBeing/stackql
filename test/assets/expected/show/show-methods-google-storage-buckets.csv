MethodName,RequiredParams
delete,bucket
get,bucket
getIamPolicy,bucket
insert,"project, data__name"
list,project
lockRetentionPolicy,"bucket, ifMetagenerationMatch"
patch,bucket
setIamPolicy,"bucket, data__bindings"
testIamPermissions,"bucket, permissions"
update,"bucket, data__acl"